nab/src/org/labkey/nab/NabAssayController.java

```diff
@@ -1261,8 +1261,8 @@ public void bindJson(JSONObject json)
                     _exclusions.add(new WellExclusion(excluded.getInt("plate"),
                         excluded.getInt("row"),

```

Comment:
Do we need optInt?

Reply:
I'm not familiar enough with this code to know what the default value would be for these two integer properties (i.e. row and col), so I'm going to leave that for now.